pictures.md

```diff
@@ -0,0 +1,145 @@
+# Picture Quality Considerations
+
+The subject should not be degraded to the point of being obscured in noise,
+or overly smoothed out, removing the detail of the subject.
+
+The background should not be degraded to the point of being distracting in the context of the subject.
+
+Recommended to avoid:
+
+- Heavy visual degradation. Drop the format if it cannot meet expected quality under 200K at 1080p.
+- Any unexpected smoothing of lines
+- Any unexpected blurring between strong colours (i.e. reddish-pink & black)
+- Any strong colour blending that shouldn't exist
+- Any muteness in colour.
+
+## The encoders in use are:
+
+- libwebp's `cwebp -preset photo -m 6 -z 9 input.png output.webp`
+- libjxl's `cjxl input.png output.jxl "--brotli_effort=11" -e 9`
+- libjxl/jpegli's `cjpegli input.png output.jpeg`
+- libavif's `avifenc -s 0 --ignore-exif --ignore-xmp input.png output.avif`
+
+All encoders are tuned to run at their slowest but highest quality possible for generic images
+to ensure that they can consistently compress into small but usable files.
+
+The pipeline in [scaler.ps1] is currently configured to:
+
+- Emit AVIF at quality 70 with YUV 4:4:4. For most scenes, this generally does well.
+- Emit JPEG at distance 1.5 with YUV 4:4:4. For nearly all scenes, this does well.
+- Emit 360p, 720p and 1080p scales of input.
+  - This allows for data saving and responsiveness as the client doesn't need to download as large of files.
```

Comment:
This is a lie; this should be AVIF q60 and JPEG d2
720p was also omitted as it wasn't deemed to be worth while; while 4K is emitted.